Example/Optik/ViewController.swift

```diff
@@ -82,4 +82,16 @@ extension ViewController: ImageViewerDelegate {
         currentLocalImageIndex = index
     }
     
+    func didTouchLikeButton(at index: Int) {
+        print(index)
```

Comment:
You are right when the user touch in the button, it's need to change the image. This code is in the example pod. So I think something like that 


` 
   func didTouchLike(button: UIButton, at index: Int) {
        if  button.currentImage == UIImage(named: "heart-full")! {
            button.setImage(UIImage(named: "heart-empty"), for: .normal)
        } else {
            button.setImage(UIImage(named: "heart-full"), for: .normal)
        }
    }
`